### ISSUE TITLE
The translator for SpringerLink works abnormal

### ISSUE THREAD

**Issue by cslalexX** [zotero/translators#3302]
SpringerLink updated their website recently, and the translator doesn't seem to work well anymore, as the details, previously, it just found a DOI of an item and used it to look up the information of this item, and now it seems just extract the information from the website, as the results, some fields of Zotero are left empty, such as authors.

**Comment by aborel** [comment]
I can't reproduce the problem, I have tried a couple of articles and chapters, everything looks OK for me. Can you provide an example URL where you experience this behavior?

**Comment by cslalexX** [comment]
Have you tested the book?
![Screenshot_20240419_230443.png](https://github.com/zotero/translators/assets/38831137/9628e8a4-aa75-4e6b-a24f-132f24d39a05)

![Screenshot_20240419_230214.png](https://github.com/zotero/translators/assets/38831137/676d3e28-802b-43ab-b8e3-e7334c448d95)

![Screenshot_20240419_230330.png](https://github.com/zotero/translators/assets/38831137/bdf9d480-49f7-4445-980c-7fd20826cac8)

**Comment by cslalexX** [comment]
And that's when you do it with DOI.

![Screenshot_20240419_230944.png](https://github.com/zotero/translators/assets/38831137/e415fb41-f6a0-4bea-9b5a-20d7e7163d6c)

**Comment by aborel** [comment]
Thanks for the details. I had not tested a book, indeed for https://link.springer.com/book/10.1007/978-3-030-43881-4 the captured metadata is rather poor. The document type, title and abstract are retrieved from embedded metadata, but apparently nothing else.
With this information, I think we can start to investigate.

**Comment by cslalexX** [comment]
> Thanks for the details. I had not tested a book, indeed for https://link.springer.com/book/10.1007/978-3-030-43881-4 the captured metadata is rather poor. The document type, title and abstract are retrieved from embedded metadata, but apparently nothing else.
> With this information, I think we can start to investigate.

Thanks for your attention, in my perspective, that problem happened probably because SpringerLink had updated their website recently and that movement caused this situation. In the older version of their website, everything works perfectly.

**Comment by aborel** [comment]
The browser connector seems to be ignoring the Springer Link translator completely for this ebook. The same thing happens for the ebook test cases from the translator code, although these tests are working as expected within the translator editor.
For ebooks, the translator is supposed to return multiple results, I'm guessing the book itself and each chapter, I don't know if that plays a role in this. I'm still looking into this issue, but if anybody with a deeper knowledge wants to offer their advice, it'll be great.

**Comment by cslalexX** [comment]
Yeah, previously when you clicked the connector button it would pop up a window and let you select what items you want to add, a whole book or some chapters, and the first one always be the whole book for the common book, for reference work, they don't have a whole book option though.

**Comment by aborel** [comment]
The problem was also reported on the Zotero forum: https://forums.zotero.org/discussion/113759/springerlink-changed-connectors-dont-work-any-more#latest

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/comment/461270/#Comment_461270

(And for future reference, there's no need to discuss icons/metadata/etc. for a case like this: you can just hover over the save icon and see that it's using the Embedded Metadata translator rather than the Springer Link translator for the new design.)
